### PR TITLE
Added new TALON for collapsed tower

### LINKED ISSUE #54239: Rocket launcher bot in collapsed tower griefs the tower and destroys quest items
### Describe the bug

Now that it can actually fire, the TALON bot will set fire to huge amounts of the tower, and will usually destroy the nanofabricator console and the quest items from the Hub01 quest, robbing the player of their rewards through no fault of their own, with no way to prevent this from happening. 
It's worth mentioning that the templates are contained inside a wooden box, the box being flammable might be responsible for the destruction of the quest templates.

### Steps to reproduce

1. Go near a collapsed tower, preferably with the Light Retrieval quest active.
2. Observe rewards being destroyed by robot.

### Expected behavior

The rocket bot should probably be replaced with something less destructive, such as CROWS or a bunch of secubots. A hugely destructive rocket bot also doesn't make sense to deploy in this location because it would destroy the tower, something obviously undesirable to its staff pre-cataclysm. If it really is just the wooden box that's causing these templates to be destroyed then simply changing it to a metal container would work, but it doesn't fix the issue of the rocket bot being incongruous.

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: 55ae5ff [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Bionic Slots [cbm_slots]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
Content "Added new TALON for collapsed tower"

#### Purpose of change
Fixes #54239 which I ran into today. Worst part was *I could see the fire and I knew what had happened*

#### Describe the solution
Reasoning: I don't think a TALON with FLASH rockets is a good fit for this area in general even if it wasn't cracking a rocket with no-one around. It seems it got changed over when the tripods got removed in a set of bulk changes without much reasoning other than "something needs to go here". I understand the value of a FLASH TALON in general for dungeons in CDDA and military strategy overall but if it was supposed to suppress biological forces and toast the place then the tower should have been ash weeks ago (within the actual chronology of the Cataclysm). There is probably a way with EOC's to force it to only spawn as you get close but **why would it only spawn then? if it's there it should be getting into fights in the belly of the beast**. I guess what I'm saying is how it's currently working is *how it really should work* which makes it a bad fit for the dungeon and quest.

So I've added created a more thematically interesting enemy with a similar lethality in the form of a Talon UGV that's been overrun with flesh and is being puppeted by the force at the tower.

It is similar to a standard TALON but its

- Even slower (crawling around on gross limbs)
- Has Some Regen (flesh carapace)
- More HP (flesh carapace) 
- Worse aim (no sensors just limbs of probably dead soldiers controlling it more directly)
- Gross description
- Default faction is zombies so no infighting

Plan is to add two of them in the finale room. They won't burn anything, they'll still be spooky. 

TODO:

- [ ] Add monster bot revert stuff
- [ ] Maybe generic guns
- [ ] Change the spawn over

#### Describe alternatives you've considered
Discussion is up above,

#### Testing
Still more to do, mainly PR'd this so people could give opinions.

#### Additional context
image

### CODE REVIEW

File: data/json/monsters/defense_bot.json [hunk 1]
```diff
@@ -140,6 +140,38 @@
       "STUN_IMMUNE"
     ]
   },
+  {
+    "id": "mon_secubot_flesh",
+    "type": "MONSTER",
+    "copy-from": "mon_secubot",
+    "name": { "str": "writhing puppet TALON UGV" },
+    "description": "This was once a TALON unmanned ground vehicle equipped with an M16A4.  However it has been completely engulfed in missmatched flesh, as it's mechanical bits attempt to move blood oozes from its every bit.  Though it's treads are non-functional a mass of fingers seems to slowly drag its hulking frame.",
```
Comment (haveric) [inline data/json/monsters/defense_bot.json:148]: Suggested tweaks:
This was once a TALON unmanned ground vehicle equipped with an M16A4. However, it has been completely engulfed in mismatched flesh.  As its mechanical bits attempt to move, blood oozes from its every bit. Though its treads are non-functional, a mass of fingers seems to slowly drag its hulking frame.

Comment (haveric) [reply data/json/monsters/defense_bot.json:148]: `completely engulfed in mismatched flesh. As its mechanical bits` could maybe be left as a single sentence, but might need to be tweaked to read better. Up to you of course

Comment (bombasticSlacks) [reply data/json/monsters/defense_bot.json]: I gave it a pass with your suggestions and beefed up the description at the end of the mechanical bits sentence to make it flow better. Gonna also swap out the double use of frame

File: data/json/monsters/defense_bot.json [hunk 2]
```diff
@@ -140,6 +140,38 @@
       "STUN_IMMUNE"
     ]
   },
+  {
+    "id": "mon_secubot_flesh",
+    "type": "MONSTER",
+    "copy-from": "mon_secubot",
+    "name": { "str": "writhing puppet TALON UGV" },
+    "description": "This was once a TALON unmanned ground vehicle equipped with an M16A4.  However, it has been completely engulfed in missmatched flesh.  As its mechanical bits attempt to move, blood oozes from its sinewy chassis.  Though it's treads are non-functional a mass of fingers seems to slowly drag its hulking frame.",
```
Comment (haveric) [inline data/json/monsters/defense_bot.json:148]: Couple more, sorry!
`missmatched` -> `mismatched`
`Though it's treads ` -> `Though its treads`
`non-functional a` -> `non-functional, a`

**Comment by bombasticSlacks** [comment]
Closed in favor of #55561

**Pull request closed without merging**